### ISSUE TITLE
Bus stop names and number in Turkey/Turkiye

### ISSUE THREAD

**Issue by Alosra** [streetcomplete/StreetComplete#4489]
Hello
I have been using this great app for contributing from Turkey but I have come across a weird thing, for bus stops the app says that bus stop code/number is never gonna be shown in Turkey. But in Antalya/Turkey, bus stops have both numbers and names and when contributing I see that bus stop number is used as the name of the bus stop.
For example there is an "Il sağılık müdürlüğü" bus stop with the bus stop number of 10163
So I searched but came across no reason for it and if there is one, I would appreciate knowing it.
But I think that bus stop number should be accessable for Turkey and hopefully a lot of quests for people to add the real name.

**Comment by westnordost** [comment]
Do you have some example photos or a source for that?

**Comment by HolgerJeromin** [comment]
> So I searched but came across no reason for it and if there is one, I would appreciate knowing it.

The stop-ref-quest uses an allow-list which is the reason why you do not find a discussion why to disallow it in turkey.

https://github.com/streetcomplete/StreetComplete/blob/3c88573043daca533b9419abb2064deddfa56646/app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_ref/AddBusStopRef.kt#L20

**Comment by Alosra** [comment]
> Do you have some example photos or a source for that?

I just took two photos of different bus stops with names and numbers.
![IMG_20221013_111928](https://user-images.githubusercontent.com/47773271/195543704-99109e05-3d1c-4089-b3ed-8e1cf897cc21.jpg)
![IMG_20221013_112020](https://user-images.githubusercontent.com/47773271/195543760-389559d5-4e34-4f1d-a916-39932c01d191.jpg)

Also there is an app named antalyakart which lets us view bus stops in antalya and which buses will come next, bus stops in there also have both name and number.

**Comment by matkoniecz** [comment]
Do you know is it the same case in other cities with bus stops? Or is it maybe limited to you city?

**Comment by Alosra** [comment]
> Do you know is it the same case in other cities with bus stops? Or is it maybe limited to your city?

I'm not sure but I know the case for my city I maybe able to research a little and find out.

**Comment by Alosra** [comment]
I searched in google maps and saw that in Istanbul and Ankara the capital bus stops are referred to by names, but when I found a website for Ankara bus system, it showed that bus stops have both names and numbers and for searching them, bus stop numbers are more convenient since they are five digit numbers. but it seems that bus stop numbers are not be unique in the country. and are more likely unique in cities or states in Turkey. Because when I searched the same bus stop number that I sent a photo of it in here, it brought another bus stop in Ankara. I should note that the website is for public transport in Ankara and it's addressed at www.ego.gov.tr/
